cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -323,6 +378,8 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                 isProctoredExam: isProctoredExam,
                 isPracticeExam: isPracticeExam,
                 isOnboardingExam: isOnboardingExam,
+                lmsUrl: lmsUrl,
+                embedLmsUrl: embedLmsUrl,
```

Comment:
```suggestion
                lmsUrl,
                embedLmsUrl
```
When keys and values are the same you can pass them and it will be the same

```
   const key = 'value'
   const object = { 'key': key };
   // same as this
   const object = { key  };
```

Reply:
I left it that way to maintain consistency with the other properties defined in the object.